gokart/config_params.py

```diff
@@ -1,23 +1,26 @@
+from typing import Dict, Optional
+
 import luigi
 
 import gokart
 
 
 class inherits_config_params:
-    def __init__(self, config_class: luigi.Config):
-        self.config_class: luigi.Config = config_class
+    def __init__(self, config_class: luigi.Config, param_config2task: Optional[Dict[str, str]] = None):
```

Comment:
@mski-iksm [nits] Plz write docstring about args here :)

Reply:
@Hi-king 
I've changed `param_config2task` to `parameter_alias`. Please review!